OmniBLE/PumpManagerUI/ViewModels/PairPodViewModel.swift

```diff
@@ -180,6 +180,28 @@ class PairPodViewModel: ObservableObject, Identifiable {
         }
     }
 
+    // Return a suitable DashPairingError for the returned PumpManagerError
+    private func mapPumpManagerError(_ pumpManagerError: PumpManagerError) -> DashPairingError {
```

Comment:
Rather than erasing the actual error and replacing it with a very vague type, you can just change the messaging.

Reply:
i.e. do this kind of switching in the method implementations for `LocalizedError`.

Reply:
I did try alternate approaches like this and ran into some difficulties in some places and gave up. I'll DM you and maybe I can to produce something else which better addresses the issues of overly verbose & detailed low level bluetooth error messages being displayed when pairing.